sale_promotion_rule/__manifest__.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Akretion (http://www.akretion.com)
+# Benoît GUILLOT <benoit.guillot@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Sale Promotion Rule",
+    "summary": "Module to manage promotion rule on sale order",
+    "version": "10.0.1.0.0",
+    "category": "Sale",
+    "website": "https://akretion.com",
+    "author": "Akretion",
```

Comment:
Missing author "Acsone, Odoo Community Association (OCA)"
```suggestion
    "author": "Akretion, Acsone, Odoo Community Association (OCA)",
```